src/sentry/static/sentry/app/views/groupUserReports.jsx

```diff
@@ -97,7 +110,7 @@ const GroupUserReports = createReactClass({
     return (
       <div className="box empty-stream">
         <span className="icon icon-exclamation" />
-        <p>{t('No user reports have been collected for this event.')}</p>
+        <p>{t('No user reports have been collected.')}</p>
```

Comment:
maybe this text should include "for this environment" when one is active?

Reply:
That sounds good to me

Reply:
I'll open a separate PR for that since there will be a bunch of other places as well, which we should align